packages/fast-component-explorer/app/explorer.style.ts

```diff
@@ -25,6 +26,24 @@ export interface ExplorerClassNameContract {
     explorer_viewerControls?: string;
 }
 
+export function applyScrollbarStyle(): CSSRules<{}> {
+    return {
+        "&::-webkit-scrollbar": {
+            background: (config: DesignSystem): string => {
+                return neutralLayerL2(config);
```

Comment:
You should just be able to pass the recipe as the value without the design system wrapper function.